core/src/epicli/data/common/ansible/playbooks/applications.yml

```diff
@@ -7,7 +7,6 @@
 
 - hosts: 127.0.0.1
   gather_facts: false
-  connection: local
```

Comment:
Is that default setting?

Reply:
So the problem/feature in 2.10 I found is that "delegate_to" does not work correctly when "connection: local" is set. I had to remove it here, because later in the playbook when we try creating posrgtres db, that delegation falls flat on its face.. :)